packages/react/test/evaluation.spec.tsx

```diff
@@ -35,369 +41,540 @@ describe('evaluation', () => {
   const REASON_ATTR_VALUE = StandardResolutionReasons.STATIC;
   const TYPE_ATTR = 'data-type';
 
-  const provider = new InMemoryProvider({
-    [BOOL_FLAG_KEY]: {
-      disabled: false,
-      variants: {
-        [BOOL_FLAG_VARIANT]: BOOL_FLAG_VALUE,
-        off: false,
+  const makeProvider = () => {
+    return new InMemoryProvider({
+      [BOOL_FLAG_KEY]: {
+        disabled: false,
+        variants: {
+          [BOOL_FLAG_VARIANT]: BOOL_FLAG_VALUE,
+          off: false,
+        },
+        defaultVariant: BOOL_FLAG_VARIANT,
       },
-      defaultVariant: BOOL_FLAG_VARIANT,
-    },
-    [STRING_FLAG_KEY]: {
-      disabled: false,
-      variants: {
-        [STRING_FLAG_VARIANT]: STRING_FLAG_VALUE,
-        parting: 'bye',
+      [STRING_FLAG_KEY]: {
+        disabled: false,
+        variants: {
+          [STRING_FLAG_VARIANT]: STRING_FLAG_VALUE,
+          parting: 'bye',
+        },
+        defaultVariant: STRING_FLAG_VARIANT,
       },
-      defaultVariant: STRING_FLAG_VARIANT,
-    },
-    [NUMBER_FLAG_KEY]: {
-      disabled: false,
-      variants: {
-        [NUMBER_FLAG_VARIANT]: NUMBER_FLAG_VALUE,
-        '2^1': 2,
+      [NUMBER_FLAG_KEY]: {
+        disabled: false,
+        variants: {
+          [NUMBER_FLAG_VARIANT]: NUMBER_FLAG_VALUE,
+          '2^1': 2,
+        },
+        defaultVariant: NUMBER_FLAG_VARIANT,
       },
-      defaultVariant: NUMBER_FLAG_VARIANT,
-    },
-    [OBJECT_FLAG_KEY]: {
-      disabled: false,
-      variants: {
-        [OBJECT_FLAG_VARIANT]: OBJECT_FLAG_VALUE,
-        empty: {},
+      [OBJECT_FLAG_KEY]: {
+        disabled: false,
+        variants: {
+          [OBJECT_FLAG_VARIANT]: OBJECT_FLAG_VALUE,
+          empty: {},
+        },
+        defaultVariant: OBJECT_FLAG_VARIANT,
       },
-      defaultVariant: OBJECT_FLAG_VARIANT,
-    },
-  });
+    });
+  };
 
-  OpenFeature.setProvider(EVALUATION, provider);
+  OpenFeature.setProvider(EVALUATION, makeProvider());
+
+  describe('rendering', () => {
```

Comment:
None of the previous tests have changed, I just formatted the file and put them under this `rendering` describe, since they are all DOM-based.